### PR TITLE
Remove extra arguments from CreateApplication

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/AspNetCoreModuleV2/OutOfProcessRequestHandler/outofprocess/outprocessapplication.h [hunk 1]
```diff
@@ -44,6 +44,16 @@ class OUT_OF_PROCESS_APPLICATION : public APPLICATION
         _Out_ IREQUEST_HANDLER   **pRequestHandler)
     override;
 
+    VOID
+    SetParameter(
```
Comment (jkotalik) [inline src/AspNetCoreModuleV2/OutOfProcessRequestHandler/outofprocess/outprocessapplication.h]: Put this in Application instead?

File: src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx [hunk 2]
```diff
@@ -260,25 +260,22 @@ BOOL APIENTRY DllMain(HMODULE hModule,
...
 }
 
-// TODO remove pHttpContext from the CreateApplication call.
 HRESULT
 __stdcall
 CreateApplication(
     _In_  IHttpServer        *pServer,
-    _In_  IHttpContext       *pHttpContext,
-    _In_  PCWSTR             pwzExeLocation,
+    _In_  IHttpApplication   *pHttpApplication,
```
Comment (jkotalik) [inline src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx]: I thought the HttpContext was used in the request handler config to check websocket support. Can you check what we do for websockets?

Comment (pakrym) [reply src/AspNetCoreModuleV2/OutOfProcessRequestHandler/dllmain.cxx]: You moved websocket check to CreateHandler where we have a context.

File: src/AspNetCoreModuleV2/OutOfProcessRequestHandler/outofprocess/outprocessapplication.h (hunk 1 above)
Comment (pakrym) [reply src/AspNetCoreModuleV2/OutOfProcessRequestHandler/outofprocess/outprocessapplication.h]: Yep

**Comment by pakrym** [comment]
Release failure: https://github.com/aspnet/IISIntegration/issues/959

**Pull request merged**